samples/ask-akka-agent/src/main/java/akka/ask/agent/api/UsersEndpoint.java

```diff
@@ -26,7 +26,7 @@ public UsersEndpoint(ComponentClient componentClient) {
   public CompletionStage<ConversationHistoryView.ConversationHistory> getSession(String userId) {
 
     return componentClient.forView()
-        .method(ConversationHistoryView::getMessagesBySession)
+        .method(ConversationHistoryView::getSessionsByUser)
```

Comment:
@autodidaddict I guess you changed this in the view, but not in the endpoint

Reply:
that's weird. I did a language server rename so I figured it would've caught it.

Reply:
Want me to push the fix to this PR?

Reply:
I fixed it in this PR. main didn't compile.